src/config.ts

```diff
@@ -129,16 +129,35 @@ export interface Configuration {
 	stagingDirInput?: string;
 }
 
+const offsetBuildNumber = (
+	buildNumber: string,
+	offset: string | undefined,
```

Comment:
Wonder if we can simplify this by defaulting offset to 0?

Reply:
yes good point. I am not a typescript expert (far from it!), can functions have default arguments?